# highlight_mode.yaml params
hl_uniform_radius: 0.01  # [m]